proto/ibc/applications/fee/v1/genesis.proto

```diff
@@ -27,6 +27,7 @@ message FeeEnabledChannel {
 message RegisteredRelayerAddress {
   string address              = 1;
   string counterparty_address = 2 [(gogoproto.moretags) = "yaml:\"counterparty_address\""];
+  string channel_id = 3 [(gogoproto.moretags) = "yaml:\"channel_id\""];
```

Comment:
Shouldn't `make proto-all` format this to float the alignment with above fields 🤔 
I'm not 100% sure tbh, minor nit either way! :)

Reply:
Just ran it again to be sure.